doc/SuppTechInfo/python/VSC-nonlinear.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Please clear out the contents of this notebook so that we aren't versioning inline figures and make sure that the cells do not produce an error.  The last cell in this notebook produces an error.

Reply:
Output in notebook cells has been cleared out in e097e0c

I commented out the last cell as I can't get that code to work on my Windoze machine.  I am hopeful that @jdhughes-usgs might be able to get the notebook to create the plot with the correct fonts (plot appears in the SuppTechInfo document)